### PR TITLE
Selectable soundpacks

### PR DESCRIPTION
**Pull request by generic-name-guy**
Yeah, the code is a mess, i might redo it later by using variables or something like that
Also, please excuse the insane amount of commits, i have no idea of how to use github correctly.

### CODE REVIEW

**Comment by mmaulwurff** [comment]
I cannot accept HL2 sounds :(
What is the license for "Alternative" sounds?

**Comment by generic-name-guy** [comment]
> I cannot accept HL2 sounds :( What is the license for "Alternative" sounds?

Welp, i might have to keep them only on my branch if that's the case, since the guy i got the alternative sounds from got them from an assortment of other Doom mods, but they could possibly be from freesound.org as well, here's hoping.

**Comment by mmaulwurff** [comment]
I'd like to keep the mod assets properly licensed. I have to make sure that sound license allows reuse, sorry.

What we can do right now - add the selectable soundpacks feature, but make sounds in them aliases for the base sounds. Then either I will find suitable sounds on freesound.org, or third-party sound-pack pk3s will be created.

**Comment by generic-name-guy** [comment]
> I'd like to keep the mod assets properly licensed. I have to make sure that sound license allows reuse, sorry.
> 
> What we can do right now - add the selectable soundpacks feature, but make sounds in them aliases for the base sounds. Then either I will find suitable sounds on freesound.org, or third-party sound-pack pk3s will be created.

Good idea, make whatever changes you need, while i look for some good sounds.

File: credits.md [hunk 1]
```diff
@@ -15,3 +15,4 @@
 - Wheel mouse input code is based on code from PyWeaponWheel by DrPyspy.
 - [MD5 Implementation in ZScript](https://github.com/3saster/GZDoom_Utilities) by 3saster (see [MD5.zs](zscript/MD5/MD5.zs) for license).
 - Brazilian portuguese localization by generic name guy.
+- hl2_tick, hl2_open, hl2_close and hl2_nope are all from [Half-Life 2](https://store.steampowered.com/app/220/HalfLife_2/), developed by Valve Software.
```
Comment (mmaulwurff) [inline credits.md:18]: Please remove non-free files and and replace them with aliases for base sounds.

**Comment by mmaulwurff** [comment]
I cannot push to your branch, you have to change sounds yourself.

**Comment by generic-name-guy** [comment]
> I cannot push to your branch, you have to change sounds yourself.

Is it fine if i create a new branch and close this pull request?

**Comment by mmaulwurff** [comment]
Do what is convenient to you, I can combine your commits into a single one, this way the unwanted files won't show up in commit history, I believe.
image

**Comment by generic-name-guy** [comment]
I am doing this because i created a new branch without the non-free sounds

**Comment by generic-name-guy** [comment]
closed this because i'm making a new one

**Pull request closed without merging**